setup.py

```diff
@@ -26,7 +31,8 @@
     packages=find_packages(),
     package_data={"tap": ["py.typed"]},
     install_requires=["typing-inspect >= 0.7.1", "docstring-parser >= 0.15"],
-    tests_require=["pytest"],
+    tests_require=test_requirements,
```

Comment:
FYI `tests_require` is deprecated since `setuptools >= 41.5.0` (October 2019) according to [this SO comment](https://stackoverflow.com/questions/4734292/specify-where-to-install-tests-require-dependencies-of-a-distribute-setuptools#comment113087361_4734292)

Reply:
We'll update it. Thank you!